jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/BuildManifestListOrSingleManifestStepTest.java

```diff
@@ -0,0 +1,157 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.builder.steps;
+
+import com.google.cloud.tools.jib.blob.BlobDescriptor;
+import com.google.cloud.tools.jib.builder.ProgressEventDispatcher;
+import com.google.cloud.tools.jib.configuration.BuildContext;
+import com.google.cloud.tools.jib.event.EventHandlers;
+import com.google.cloud.tools.jib.image.Image;
+import com.google.cloud.tools.jib.image.Layer;
+import com.google.cloud.tools.jib.image.json.ManifestTemplate;
+import com.google.cloud.tools.jib.image.json.V22ManifestListTemplate;
+import com.google.cloud.tools.jib.image.json.V22ManifestTemplate;
+import java.io.IOException;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/** Tests for {@link BuildManifestListOrSingleManifest}. */
+@RunWith(MockitoJUnitRunner.class)
+public class BuildManifestListOrSingleManifestStepTest {
+
+  @Mock private ProgressEventDispatcher.Factory progressDispatcherFactory;
+  @Mock private BuildContext buildContext;
+  @Mock private Layer layer;
+
+  private Image image1;
+  private Image image2;
+
+  @Before
+  public void setUp() {
+    Mockito.when(buildContext.getEventHandlers()).thenReturn(EventHandlers.NONE);
+    Mockito.doReturn(V22ManifestTemplate.class).when(buildContext).getTargetFormat();
+    Mockito.when(layer.getBlobDescriptor()).thenReturn(new BlobDescriptor(0, null));
+
+    image1 =
+        Image.builder(V22ManifestTemplate.class)
+            .setArchitecture("amd64")
+            .setOs("linux")
+            .addLayer(new PreparedLayer.Builder(layer).setName("resources").build())
+            .build();
+    image2 =
+        Image.builder(V22ManifestTemplate.class)
+            .setArchitecture("arm64")
+            .setOs("ubuntu")
+            .addLayer(new PreparedLayer.Builder(layer).setName("classes").build())
+            .build();
+  }
+
+  @Test
+  public void testCall_singleManifest() throws IOException {
+    /**
+     * Expected manifest JSON:
+     *
+     * <pre>{@code
+     * {
+     * "schemaVersion":2,
+     * "mediaType":"application/vnd.docker.distribution.manifest.v2+json",
+     * "config":{
+     * "mediaType":"application/vnd.docker.container.image.v1+json",
+     * "digest":"sha256:1b2ff280940537177565443144a81319ad48528fd35d1cdc38cbde07f24f6912",
+     * "size":158
+     * },
+     * "layers":[
+     * {
+     * "mediaType":"application/vnd.docker.image.rootfs.diff.tar.gzip",
+     * "size":0
+     * }
+     * ]
+     * }
+     * }</pre>
+     */
```

Comment:
Javadoc comments (starting with `/**` using double asterisks) are basically for machine processing to generate a nice-looking manual for external users using a library. This is an internal code comment, so we should use `// code comment`. And no HTML formatting, and no need to follow the Javadoc convention. I also suggest proper indentation for readability.

```
// Expected manifest JSON:
// {
//   "schemaVersion":2,
//   "mediaType":"application/vnd.docker.distribution.manifest.v2+json",
```

Reply:
Yes was the issue I was asking about in the meeting.I'l remove the java doc format and use code comments instead.